### DISCUSSION TITLE
Git could not be found at the expected path

### DISCUSSION THREAD

**Discussion by spottedmahn** [community/community#21906] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/2b81a0ad0c3df9039efd23e0e873f03fb1a67dc1.png" data-download-href="/uploads/short-url/6cSfWyo47jTkWPn7QC8NSRmKj0B.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/14079228/181067489-c4a3ffcb-f66c-4357-87d3-75c42f74cb99.png" alt="image.png" data-base62-sha1="6cSfWyo47jTkWPn7QC8NSRmKj0B" width="690" height="411" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/2b81a0ad0c3df9039efd23e0e873f03fb1a67dc1_2_690x411.png, https://aws1.discourse-cdn.com/github/original/1X/2b81a0ad0c3df9039efd23e0e873f03fb1a67dc1.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/2b81a0ad0c3df9039efd23e0e873f03fb1a67dc1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/2b81a0ad0c3df9039efd23e0e873f03fb1a67dc1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">1010×602 49.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>‘C:\Users\mdepouw\AppData\Local\GitHubDesktop\app-1.0.11\resources\app\git\cmd\git.exe’. This might be a problem with how the application is packaged, so confirm this folder hasn’t been removed when packaging.</p>

**Comment by hcchengithub** [comment, marked as answer]
<p>The answer <a href="https://github.com/desktop/desktop/issues/4165" rel="nofollow noopener">https://github.com/desktop/desktop/issues/4165</a> works</p>
